services/gateway/apis/http-version3/swagger/definitions/nft.json

```diff
@@ -1,4 +1,104 @@
 {
+    "NFTAttributes": {
+        "type": "object",
+        "required": [
+            "module",
+            "attributes"
+        ],
+        "properties": {
+            "module": {
+                "type": "string",
+                "example": "customModule"
+            },
+            "attribute": {
+                "type": "string",
+                "example": "Custom attribute"
+            }
+        }
+    },
+    "NFT": {
+        "type": "object",
+        "required": [
+            "id",
+            "nft",
+            "owner",
+            "attributes",
+            "lockingModule"
+        ],
+        "properties": {
+            "id": {
+                "type": "string",
+                "description": "Universal identifier of the NFT.",
+                "example": "00000000100000001"
+            },
+            "nft": {
+                "type": "object",
+                "required": [
+                    "chainID",
+                    "collectionID",
+                    "index"
+                ],
+                "properties": {
+                    "chainID": {
+                        "type": "string",
+                        "description": "Chain ID on which the NFT was created.",
+                        "example": "00000000"
+                    },
+                    "collectionID": {
+                        "type": "string",
+                        "description": "Collection ID of the NFT.",
+                        "example": "10000000"
+                    },
+                    "index": {
+                        "type": "number",
+                        "description": "Index of NFT in the collection.",
+                        "example": 1
+                    }
+                }
+            },
+            "owner": {
+                "type": "string",
+                "description": "owner of the NFT. It can either be an address or chain ID.",
```

Comment:
```suggestion
                "description": "Owner of the NFT. It will be an address when owned by an account on the chain it was created or a chainID when escrowed.",
```